### DISCUSSION TITLE
Is there a way to send a custom email?

### DISCUSSION THREAD

**Discussion by steelzeh** [pocketbase/pocketbase#1321] [Q&A] [answered]
Hi guys, is there a way to send a custom email, that is not 1 of the 3 templates.

Let say once there is a creation in a custom collection, i would like to send a custom email to one or multiple recipients?
If there isn't, is there a way i would be able to create that functionality my selft?

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment the only way to send custom email is to [use PocketBase as framework](https://pocketbase.io/docs/use-as-framework/). You can define your own trigger (endpoint, cli command, event hook, etc.) and use the `app.NewMailClient()` instance:

```go
app.NewMailClient().Send(&mailer.Message{
  From: from,
  To: to,
  Subject: subject,
  HTML: html,
  // additional configurable fields
  Attachments: attachments,
  Bcc: []string{"bcc1@example.com", "bcc2@example.com"},
  Cc: []string{"cc1@example.com", "cc2@example.com"},
  Headers: map[string]string{"Custom-Header": "test"},
  Text: "custom plain text version",
})
```

You can also check #317.

**Reply by steelzeh** [reply]
Thanks for fast answer! Works
